### DISCUSSION TITLE
Turf > 7.0.0

### DISCUSSION THREAD

**Discussion by coracuity** [Turfjs/turf#2681] [Q&A]
Hey y'all, thanks so much for the awesome project! It's been a huge help in personal and professional projects and is hands down one of my favorite libraries to work with.

I was wondering what the release cadence might be like with Turf? There's at least one fix on master that I would loooooove to get my hands on, even as an alpha release.

Again, thanks so much for the cool library!!

**Comment by twelch** [comment]
You should be able to see the release cadence on NPM, it is sporadic, and sometimes few and far between.  There was a 7.1 prerelease about a month or two ago.  Might be another release in the next couple couple weeks, but no promises.
